test/images/sample-apiserver/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
You don't need to have `Makefile` if you don't have any source files to compile. In this case you are getting the code and building it in Dockerfile anyways, can you just remove this `Makefile`?

Reply:
Done!